homeassistant/components/whirlpool/quality_scale.yaml

```diff
@@ -0,0 +1,81 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: |
+      The integration does not provide any additional actions.
+  appropriate-polling: done
```

Comment:
Would it make sense to move `_attr_should_poll = False` to the base entity and only overwrite it on the entity that needs it?

Reply:
Done: https://github.com/home-assistant/core/pull/143100